packages/glibc_build237.rb

```diff
@@ -0,0 +1,323 @@
+    system "sed -i 's,/usr/sbin/tzdata-update,/bin/true,g' fedora/build-locale-archive.c"
+    system "sed -i 's,verbose,locale_verbose,g' fedora/build-locale-archive.c"
+    system "sed -i 's,be_quiet,locale_be_quiet,g' fedora/build-locale-archive.c"
+
+    @googlesource_branch = 'release-R123-15786.B'
+    system "git clone --depth=1 -b  #{@googlesource_branch} https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay googlesource"
+    Dir.glob('googlesource/sys-libs/glibc/files/local/glibc-2.37/*.patch').each do |patch|
+      puts "patch -Np1 < #{patch} || true" if @opt_verbose
+      system "patch -Np1 -F 10 -i #{patch} || true"
+    end
+  end
+
+  def self.build
+    FileUtils.mkdir_p 'glibc_build'
+    Dir.chdir 'glibc_build' do
+      if @libc_version.to_f >= 2.32
+        # Optimization flags from https://github.com/InBetweenNames/gentooLTO
+        case ARCH
+        when 'armv7l', 'aarch64'
+          system "CFLAGS='-fuse-ld=mold -pipe -O2 -fipa-pta \
+          -fno-semantic-interposition -fdevirtualize-at-ltrans' \
+             ../configure \
+            --prefix=#{CREW_PREFIX} \
+            --libdir=#{CREW_LIB_PREFIX} \
+            --with-headers=#{CREW_PREFIX}/include \
+            --without-gd \
+            ac_cv_header_cpuid_h=yes \
+            ac_cv_lib_audit_audit_log_user_avc_message=no \
+            ac_cv_lib_cap_cap_init=no \
+            --disable-profile \
+            --disable-sanity-checks \
+            --disable-werror \
+            --enable-bind-now \
+            --enable-hacker-mode \
+            --enable-kernel=5.10 \
+            --enable-shared \
+            libc_cv_386_tls=yes \
+            libc_cv_arm_tls=yes \
+            libc_cv_asm_cfi_directives=yes \
+            libc_cv_broken_visibility_attribute=no \
+            libc_cv_c_cleanup=yes \
+            libc_cv_forced_unwind=yes \
+            libc_cv_gcc___thread=yes \
+            libc_cv_hashstyle=no \
+            libc_cv_mlong_double_128ibm=yes \
+            libc_cv_mlong_double_128=yes \
+            libc_cv_predef_fortify_source=no \
+            libc_cv_visibility_attribute=yes \
+            libc_cv_x86_64_tls=yes \
+            libc_cv_z_combreloc=yes \
+            libc_cv_z_execstack=yes \
+            libc_cv_z_initfirst=yes \
+            libc_cv_z_nodelete=yes \
+            libc_cv_z_nodlopen=yes \
+            libc_cv_z_relro=yes \
+            --with-binutils=binutils \
+            --with-bugurl=https://github.com/chromebrew/chromebrew/issues/new \
+            --without-cvs \
+            --without-selinux \
+            "
+          # install-symbolic-link segfaults on armv7l, but we're deleting
+          # the libraries anyways, so it doesn't matter.
+          system "sed -i 's,install-symbolic-link,/bin/true,g' ../Makefile"
+          system "sed -i 's,symbolic-link-prog := $(elf-objpfx)sln,symbolic-link-prog := /bin/true,g' ../Makerules"
```

Comment:
Add to `def self.patch`?

Reply:
I don't think that works. These files are generated by autotools/configure...

Reply:
Fair enough.